### PR TITLE
Did someone say SERVER BROWSER??

### LINKED ISSUE #48: Server browser logic
Menu is separate (#34).

- A server can register itself by talking to the server browser.
- The server browser periodically sends information about servers to connected clients (every second?). No client -> server communication is needed (?).
- Clients "switch" their connection from the server browser to the new server.

### PR DESCRIPTION
**Pull request by sornas**
In practice it's very flimsy but the groundwork is done!

### CODE REVIEW

File: browser.sy [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+
+set_servers :: fn recv_servers: [Server] {
+    servers = recv_servers
+}
+
 start :: fn {
     n_rpc_start_server' 8388
     loop {
         n_rpc_resolve'
+        print len(servers)
```
Comment (EMattfolk) [inline browser.sy]: Maybe remove the print

**Review by EMattfolk — APPROVED** [review]
pog

File: sylt/sylt-std/src/sylt.rs [hunk 2]
```diff
@@ -41,6 +41,29 @@ pub fn for_each(ctx: RuntimeContext<'_>) -> Result<Value, RuntimeError> {
...
+    match values.as_ref() {
+        [Value::List(list), callable] => {
+            let list = Rc::clone(list);
+            let callable = callable.clone();
+            let mapped = list
+                .borrow()
+                .iter()
+                .map(|element| ctx.machine.eval_call(callable.clone(), &[element]).unwrap())
+                .collect();
+            return Ok(Value::List(Rc::new(RefCell::new(mapped))));
```
Comment (FredTheDino) [inline sylt/sylt-std/src/sylt.rs:56]: <3

**Review by FredTheDino — APPROVED** [review]
LGTM

**Pull request merged**